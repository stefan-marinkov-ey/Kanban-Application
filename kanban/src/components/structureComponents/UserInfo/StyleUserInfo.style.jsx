import styled from "styled-components";

export const StyleUserInfo = styled.div`
  width: 90%;
  display: flex;
  justify-content: space-between;
  margin: 0 auto;
  .nameOfBoard {
    width: 20%;
    h3 {
      font-size: large;
    }
  }
  .userData {
    width: 40%;
    display: flex;
    justify-content: space-between;
    p {
      width: 30%;
      margin-left: 50%;
      @media (max-width: 869px) {
        margin-left: 30%;
        font-size: smaller;
      }
      @media (max-width: 600px) {
        margin-left: 0;
        font-size: small;
      }
    }
    img {
      display: block;
      @media (max-width: 600px) {
        margin-left: 0;
        font-size: small;
        width: 40%;
      }
    }
  }
`;
