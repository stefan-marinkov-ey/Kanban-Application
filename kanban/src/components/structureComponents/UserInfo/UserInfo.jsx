import React from "react";
import { welcome } from "../../../utility/constantsText";
import { StyleUserInfo } from "./StyleUserInfo.style";

const UserInfo = ({ nameBoard, user, picture }) => {
  return (
    <StyleUserInfo>
      <div className="nameOfBoard">
        <h3>{nameBoard}</h3>
      </div>
      <div className="userData">
        <p>
          {welcome}
          <br /> {user}
        </p>
        <img src={picture} alt={user} />
      </div>
    </StyleUserInfo>
  );
};

export default UserInfo;
