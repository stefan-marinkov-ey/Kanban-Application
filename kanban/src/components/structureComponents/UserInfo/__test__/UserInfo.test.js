import React from "react";
import { shallow } from "enzyme";
import UserInfo from "../UserInfo";

describe("UserInfo", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<UserInfo />);
  });
  it("render component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
