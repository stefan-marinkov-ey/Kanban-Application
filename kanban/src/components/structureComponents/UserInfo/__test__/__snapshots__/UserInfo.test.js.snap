// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo render component 1`] = `ShallowWrapper {}`;
